src/NLog/Targets/DatabaseTarget.cs

```diff
@@ -260,11 +263,42 @@ public DatabaseTarget(string name) : this()
         [DefaultValue(CommandType.Text)]
         public CommandType CommandType { get; set; }
 
+        /// <summary>
+        /// Gets or sets property name of the SQL command parameter to set parameter DbType.
+        /// </summary>
+        /// <remarks>
+        /// May set strong DbType, for SQL Server is SqlDbType.
+        /// </remarks>
+        /// <docgen category='SQL Statement' order='12' />
+        [DefaultValue("DbType")]
+        public string ParameterDbTypePropertyName { get; set; }
+
+        /// <summary>
+        /// Gets or sets converter type of the SQL command parameter value.
+        /// </summary>
+        /// <docgen category='SQL Statement' order='13' />
+        [DefaultValue(typeof(DatabaseParameterTypeSetter))]
+        public Type ParameterConverterType { get; set; }
```

Comment:
@snakefoot any opinion on this? It's injecting the (factory) type as prop and I think this (could) work from XML config.

Reply:
No idea. I would think that people who need custom ParameterConverterType would inherit from DataBaseTarget and inject them directly.

Reply:
OK thx, I removed this construction for now. It's nice, but not sure if it's better